lib/package.json

```diff
@@ -79,28 +84,29 @@
     "babel-preset-latest": "^6.24.1",
     "classnames": "^2.2.5",
     "cross-env": "^5.1.0",
+    "date-fns": "^2.0.0-alpha.7",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.0",
-    "eslint": "^4.17.0",
+    "eslint": "^4.18.1",
     "eslint-config-airbnb": "^16.0.0",
-    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-import": "^2.9.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
-    "eslint-plugin-react": "^7.5.1",
+    "eslint-plugin-react": "^7.7.0",
     "fs-extra": "^4.0.2",
     "glob": "^7.1.2",
     "jest": "^21.2.1",
-    "material-ui": "^1.0.0-beta.33",
+    "material-ui": "^1.0.0-beta.35",
```

Comment:
Are all breaking changes of mui 34 already resolved? If so, mui needs to be updated in `peerDependencies` too

Reply:
also, there are no breaking changes in mui 35, so this release will support mui 34 as well

Reply:
We have not Switched, checkbox and radios so there were not breaking changes for us :)

Reply:
nice ;)

Reply:
I have updated dependency, not the peer dependency here.